src/domain/event/eventCategorisation/EventCategorisation.tsx

```diff
@@ -58,7 +72,14 @@ const EventCategorisation: React.FC<Props> = ({ eventData, language }) => {
       <div className={styles.priceRow}>
         <div>
           <TextTitle>{t('eventDetails.categorisation.labelPrice')}</TextTitle>
-          <p>{t('eventDetails.categorisation.eventIsFree')}</p>
+          {isFree && <p>{t('eventDetails.categorisation.eventIsFree')}</p>}
+          {!isFree && (
+            <p className={styles.eventPriceMultiline}>
+              {price}
+              {priceDescription !== '' && `\n${priceDescription}`}
+              {priceInfoUrl !== '' && createLink(`\nInfo: `, priceInfoUrl)}
```

Comment:
these can be changed then (little bit cleaner):

```
   {priceDescription && `\n${priceDescription}`}
   {priceInfoUrl && createLink(`\nInfo: `, priceInfoUrl)}
```

Reply:
Changed